pslab/instrument/oscilloscope.py

```diff
@@ -132,6 +132,10 @@ def capture(
 
         self._check_args(channels, samples, timegap)
         timegap = int(timegap * 8) / 8
+        
```

Comment:
Codacy is complaining about trailing white space on this line. Please remove it.

Reply:
Done. Also, in CONTRIBUTING.md line 65 is a bit confusing. (😅 if you could add a comment to this thread just for me to understand that better. Thank You)